examples/embedding.c

```diff
@@ -74,8 +74,15 @@ int main()
         // define julia function and call it
 
         jl_eval_string("my_func(x) = 2*x");
-
         jl_function_t *func = jl_get_function(jl_current_module, "my_func");
+        
+        // if "my_func" is in a file named "mymodule.jl",
+        // make sure to put it in a module with the same name as the file, i.e. "mymodule",
+        // then replace the above two statements with the three statements commented here below:
+        // jl_load("mymodule.jl");
+        // jl_value_t * mod = (jl_value_t*)jl_eval_string("mymodule");
```

Comment:
Sorry, what do you mean by rooted?

On Friday, September 9, 2016, Yichao Yu <notifications@github.com> wrote:

> In examples/embedding.c
> <https://github.com/JuliaLang/julia/pull/18422#discussion_r78182735>:
>
> >          jl_function_t *func = jl_get_function(jl_current_module, "my_func");
> > +
> > +        // if "my_func" is in a file named "mymodule.jl",
> > +        // make sure to put it in a module with the same name as the file, i.e. "mymodule",
> > +        // then replace the above two statements with the three statements commented here below:
> > +        // jl_load("mymodule.jl");
> > +        // jl_value_t * mod = (jl_value_t*)jl_eval_string("mymodule");
>
> This should in principle be rooted.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaLang/julia/pull/18422/files/37ec5b0f95da921fe8b885d3204ddf7d6be5fc39#r78182735>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF97wW7cW5HI07gxvGBvLU7FNQwSqfKyks5qoWQvgaJpZM4J5D5j>
> .
>